baseband/gsb/frame.py

```diff
@@ -36,15 +36,15 @@ class GSBFrame(VLBIFrameBase):
     @classmethod
     def fromfile(cls, fh_ts, fh_raw, payloadsize=1 << 24, nchan=1, bps=4,
                  complex_data=False, valid=True, verify=True):
-        """Read a frame from timestamp and raw data file handles.
+        """Read a frame from timestamp and raw data filehandles.
```

Comment:
I think removing the space is OK, though note that the formal python term for this appears to be "file object"...

Reply:
This was inconsistent throughout the docstrings (which used to either use `filehandle` or `file handle`), so I just removed whichever was less popular.